danielMunteanu/src/main/java/danielCollectionTask/Main.java

```diff
@@ -0,0 +1,66 @@
+package danielCollectionTask;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+class Main {
+    public static void main(String[] args) {
+
+        Address address1 = new Address("Moldova", "Chisinau", "Decebal");
+        Address address2 = new Address("Italy", "Roma", "Padre");
+
+        Map<String, Address> addressMapOne = new HashMap();
+        addressMapOne.put("Home", address1);
+        addressMapOne.put("Work", address2);
+
+        Map<String, Address> addressMapTwo = new HashMap();
+        addressMapTwo.put("Home", new Address("Romania", "Bucurest", "Stefan cel Mare"));
+
+        User user1 = new User("1", "Daniel", addressMapOne);
+        User user2 = new User("2", "Egor", addressMapTwo);
+
+        List<User> usersList = new ArrayList<>();
+        usersList.add(user1);
+        usersList.add(user2);
+
+        List<Item> itemList1 = new ArrayList<>();
+        Item item = new Item("telefon", Type.CLOTHES, 189.12);
+        Item item1 = new Item("bike", Type.TOY, 150.65);
+        itemList1.add(item);// add item to list
+        itemList1.add(item1);
+
+        List<Item> itemList2 = new ArrayList<>();
+        Item item2 = new Item("nexus", Type.TOY, 346.96);
+        Item item3 = new Item("nexus", Type.TOY, 79.99);
+        itemList2.add(item2);
+        itemList2.add(item3);
+
+        UserOrder userOrderOne = new UserOrder(itemList1);
+        userOrderOne.setItems(itemList1);
+        userOrderOne.getDate();
+
+        UserOrder userOrderTwo = new UserOrder(itemList2);
+        userOrderTwo.setItems(itemList2);
+        userOrderTwo.getDate();
+
+        List<UserOrder> userOrderList = new ArrayList<>();
+        userOrderList.add(userOrderOne);
+        userOrderList.add(userOrderTwo);
+
+        //Filter Orders by TOY, filterOrderByItemType(orderList)
+        List<UserOrder> filteredUserOrderList = FilterByType.filterOrderByItemType((userOrderList));
+        System.out.println("the size of filtered order list is " + filteredUserOrderList.size());
+
+        List<User> filteredListofUsers;
+        filteredListofUsers = FilterByCity.filterUsersByCity(usersList, "Chisinau"); // call for viewing the users from Chisinau city
+        System.out.println("The filtered list of users is " + filteredListofUsers.size());
+
+        System.out.println("Sum of order two is " + userOrderTwo.sum());
+        System.out.println("Count or order two is : " + userOrderTwo.count());
+        System.out.println("Date of order one is : " + userOrderOne.date());
+        System.out.println("nr iteme identice " + userOrderTwo.count(item3));
+
```

Comment:
can not see the call for sum(item3) method

Reply:
added